pom.xml

```diff
@@ -1878,20 +1878,44 @@ org.hibernate.search.spi.SearchIntegratorBuilder#buildSearchIntegrator()
                 <!-- Asciidoctor and Karaf won't work on JDK9 yet  -->
                 <jdk>1.8</jdk>
             </activation>
+            <properties>
+                <!-- Surefire versions are a minefield of bugs, so depending on context we need to choose a different one -->
+                <maven.plugin.surefire.version>2.20.1</maven.plugin.surefire.version>
+                <maven.plugin.failsafe.version>2.20.1</maven.plugin.failsafe.version>
+            </properties>
             <modules>
                 <module>documentation</module>
             </modules>
         </profile>
 
         <profile>
-            <id>jigsaw</id>
+            <id>jdk9</id>
```

Comment:
Ok if I rename the "non-jigsaw" profile to jdk8 then?

Reply:
ok